control/protocol.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: 3.0.3
+info:
+  title: Control protocol models
+  description: >-
+    All control protocol schemas
+  version: ^0.0.0
+components:
+  schemas:
+    State.Protocol.All:
+      description: >-
+        Sets all configured protocols to `start` or `stop` state.
+        
+        Setting protocol state to `start` shall be a no-op if preceding `set_config` API call
```

Comment:
Should it also result in a warning?

Reply:
Indeed, I've opened a separate issue to document warnings in other places too.

https://github.com/open-traffic-generator/models/issues/309